randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+            "Gravity Suit&main-color=#89D6FF; to jump off of a raised stone in the lava near the far ledge.\n"
+            "A similar method uses a double bomb jump from the same location, though it requires not having the "
+            "Gravity Suit.\nA harder method uses &main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; to scan-dash"
+            " off of pickups left by the crates.\n\nFinally, it is possible to get across with "
+            "&main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; by L-jumping between the lips of the "
+            "leftmost pillars.\nThis is more difficult than it appears, as the closest pillar is not standable."
+        ),
+    ],
+    "455264308": [
+        # No Grappling/Spidering Thru Damage Tip
+        # Replaces Meditation Fountain panel scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Taking damage will interrupt use of the Spider Ball or Grapple Beam. "
+            "This includes damage from superheated rooms."
+        ),
+    ],
+    "2410883438": [
+        # Just Hold L Tip
+        # Replaces Main Plaza bird image scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Holding &image=SA,3.0,1.0,1.0,46434ED3,34E79314; grants greater "
+            "control over aerial movement, making precise jumps easier."
+        ),
+    ],
+    "389886815": [
+        # Just Tap B Tip
+        # Replaces Observatory left computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;When performing a scan dash or lock-on dash, jump by tapping "
+            "&image=SI,0.70,0.68,833BEE04; for as little time as possible to gain extra height."
+        ),
+    ],
+    "3668253850": [
+        # Item Drop Tip
+        # Replaces Waterfall Cavern hole scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Each room has different pickups that drop from crates and defeated enemies.\n\n"
+            "If a specific pickup does not seem to be appearing (e.g. Power Bomb ammo), try a different room."
+        ),
+    ],
+    "3673769160": [
+        # L-Jump Frames Tip
+        # Replaces Elite Research top floor computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Holding &image=SA,3.0,1.0,1.0,46434ED3,34E79314; for the first 13 frames of a jump "
+            "and then letting go\nslightly extends jumps. This is called an L-Jump. When using the &main-color=#FF3333;"
+            "Space Jump Boots&main-color=#89D6FF;, holding &image=SA,3.0,1.0,1.0,46434ED3,34E79314; "
+            "for 7 frames after the second jump grants an additional, smaller extension."
```

Comment:
This is an arbitrary location to put this scan. It would be much more suitable to put information about doing this spinner without boost instead.

Reply:
Do you have a suggestion for where better to put L-jump advice? (Also, I don't know much about boostless spinner use, so I'm not sure what advice would be best there.)